pywhy_graphs/algorithms/pag.py

```diff
@@ -250,7 +251,7 @@ def discriminating_path(
         # check distance criterion to prevent checking very long paths
         distance += 1
         if distance > 0 and distance > max_path_length:
-            logger.warn(
+            warn(
```

Comment:
I think I would not change this. See [here](https://stackoverflow.com/a/14762106), why:
> warnings.warn() in library code if the issue is avoidable and the client application should be modified to eliminate the warning
>
> logging.warning() if there is nothing the client application can do about the situation, but the event should still be noted

I think the latter applies here, no?